subsys/bluetooth/audio/ascs.c

```diff
@@ -1163,6 +1175,7 @@ static struct bt_ascs_ase *ase_new(struct bt_ascs *ascs, uint8_t id)
 
 done:
 	ase_init(ase, i);
+	audio_iso_init(audio_iso);
```

Comment:
I think that `audio_iso_get_or_new` is better place to init the structure members

Reply:
That is better, thanks. 

Alternatively, since are just assigning constant value, we could do it within a `SYS_INIT`.